defaults/test_default.yaml

```diff
@@ -231,3 +231,25 @@ simulated_racks: 0
 use_dns_names: false
 
 use_placement_group: false
+
+scylla_network_config:
+- address: listen_address
+  ip_type: ipv4
+  public: false
+  use_dns: false
+  nic: 0
+- address: rpc_address
+  ip_type: ipv4
+  public: false
+  use_dns: false
+  nic: 0
+- address: broadcast_rpc_address
+  ip_type: ipv4
+  public: false
+  use_dns: false
+  nic: 0
+- address: broadcast_address
+  ip_type: ipv4
+  public: true
```

Comment:
why it's public ?

Reply:
removed this definition from `test_default.yaml` and changed to `false`